ci-operator/jobs/openshift/ci-ns-ttl-controller/OWNERS

```diff
@@ -2,7 +2,6 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-- alexnpavel
```

Comment:
This is suspicious... A mistake in https://github.com/openshift/ci-tools/pull/493, or does Alex have a misconfigured account? @alvaroaleman FYI

/hold

Reply:
Good catch, both https://api.github.com/users/AlexNPavel and https://api.github.com/users/alexnpavel return the correct result, so I guess we have to lowercase everything before we compare. Are you aware of any other quirks here_

Reply:
Yeah, GH usernames are apparently case insensitive. I know of no other quirks like that.

Reply:
Looks right to me now

Reply:
/hold cancel